### DISCUSSION TITLE
How to use .env variables inside of the one component?

### DISCUSSION THREAD

**Discussion by ricar2ce** [quasarframework/quasar#17022] [General - Components / Directives / etc]
How to use .env variables inside of the one component using composition api width script setup, so that it works in production mode.

thank u very much

**Comment by Dtsiantaris** [comment]
`process.env.VAR_NAME`
I would suggest using dotenv. 
Have a look at this:
https://quasar.dev/quasar-cli-webpack/handling-process-env/#using-dotenv
You might stumble on an error about `globalThis` being undefined. Solution while using app-vite is adding this line:
```
rawDefine: {
        'globalThis.process.env.NODE_ENV': JSON.stringify(process.env.NODE_ENV)
 },
```
inside the build of quasar config.
